### DISCUSSION TITLE
Question regarding starter template

### DISCUSSION THREAD

**Discussion by hugo-sid** [imfing/hextra#72] [Q&A] [answered]
[Hextra starter template](https://github.com/imfing/hextra-starter-template) doesn't have a `.devcontainer/devcontainer.json` yet.

But when I launch it in [GitHub Codespaces](https://github.com/codespaces) and run `hugo version`, I see the following output:

```
hugo v0.118.2-da7983ac4b94d97d776d7c2405040de97e95c03d linux/amd64 BuildDate=2023-08-31T11:23:51Z VendorInfo=gohugoio
```

Hugo is not in the list of [preinstalled programs](https://github.com/devcontainers/images/tree/main/src/universal#linux-universal-image) in the Codespace Ubuntu image.

How is the latest Hugo version is getting installed?

**Comment by imfing** [comment, marked as answer]
It's installed via the `.devcontainer/devcontainer.json` in the link you posted

https://github.com/devcontainers/images/blob/98e790445bd1c3ec93a5e6fa25b02e9662da6aee/src/universal/.devcontainer/devcontainer.json#L17

**Reply by hugo-sid** [reply]
Thanks for pointing it out @imfing.
